configuration.rst

```diff
@@ -304,93 +214,18 @@ There is also a ``.env`` file which is loaded. Its contents become environment v
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/doctrine
-                http://symfony.com/schema/dic/doctrine/doctrine-1.0.xsd">
-
-            <doctrine:config>
-                <doctrine:dbal
-                    driver="pdo_mysql"
-
-                    user="%database_user%"
-                    password="%database_password%" />
-            </doctrine:config>
-        </container>
-
-    .. code-block:: php
-
-        // app/config/config.php
-        $container->loadFromExtension('doctrine', array(
-            'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                // ...
-
-                'user'     => '%database_user%',
-                'password' => '%database_password%',
-            ),
-        ));
-
-But the ``parameters.yml`` file *is* special: it defines the values that usually
-change on each server. For example, the database credentials on your local
-development machine might be different from your workmates. That's why this file
-is not committed to the shared repository and is only stored on your machine.
-
-Because of that, **parameters.yml is not committed to your version control**. In fact,
-the ``.gitignore`` file that comes with Symfony prevents it from being committed.
-
-However, a ``parameters.yml.dist`` file *is* committed (with dummy values). This file
-isn't read by Symfony: it's just a reference so that Symfony knows which parameters
-need to be defined in the ``parameters.yml`` file. If you add or remove keys to
-``parameters.yml``, add or remove them from ``parameters.yml.dist`` too so both
-files are always in sync.
-
-.. sidebar:: The Interactive Parameter Handler
-
-    When you :ref:`install an existing Symfony project <install-existing-app>`, you
-    will need to create the ``parameters.yml`` file using the committed ``parameters.yml.dist``
-    file as a reference. To help with this, after you run ``composer install``, a
-    Symfony script will automatically create this file by interactively asking you
-    to supply the value for each parameter defined in ``parameters.yml.dist``. For
-    more details - or to remove or control this behavior - see the
-    `Incenteev Parameter Handler`_ documentation.
+The ``.env`` file is special, because it defines the values that usually change
+on each server. For example, the database credentials on your local development
+machine might be different from your workmates. That's why this file is **not
+committed to the shared repository** and is only stored on your machine. In
+fact, the ``.gitignore`` file that comes with Symfony prevents it from being
+committed.
+
+However, a ``.env.dist`` file *is* committed (with dummy values). This file
+isn't read by Symfony: it's just a reference so that Symfony knows which
+parameters need to be defined in the ``.env.dist`` file. If you add or remove
```

Comment:
`need to be defined in the ``.env.dist`` file` -> `need to be defined in the ``.env`` file` ?